### PR TITLE
7932 mhv maintenance windows

### PR DESCRIPTION
**Pull request by kpethtel**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
Adjusts our maintenance windows based on [this chart](https://dsva.slack.com/archives/C018V2JCWRJ/p1706803015085569?thread_ts=1706802786.997869&cid=C018V2JCWRJ). Additionally, Stephen Barrs told me `THe intent is that mhv_meds will turn off prescription functionality.  mhv_mr will turn off other medical record data, such as labs/tests, vitals, vaccines, allergies.  Note that because Mobile gets vaccines from Lighthouse with their own vets-api integration, this would not apply to vaccines in mobile app.`

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7932

## Testing done
Specs pass. Will test endpoint in staging. 

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb [hunk 1]
```diff
@@ -24,10 +24,13 @@ class MaintenanceWindowsController < ApplicationController
         %i[lighthouse disability_rating],
         %i[lighthouse letters_and_documents],
         %i[lighthouse immunizations],
```
Comment (kpethtel) [inline modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb:26]: So there are four mhv related maintenance windows we're looking for: mhv, mhv platform, mhv meds, and mhv sm. If mhv or mhv platform is down, then rx and secure messaging are down.

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by Tonksthebear — APPROVED** [review]
LGTM, I'll rebase my branch on this

**Review by aherzberg — APPROVED** [review]
LGTM

**Pull request merged**